server.js

```diff
@@ -106,11 +107,89 @@ app.use(session({
 // Add variables that are available in all views
 app.locals.analyticsId = analyticsId
 app.locals.asset_path = '/public/'
+app.locals.useAutoStoreData = (useAutoStoreData === 'true')
 app.locals.cookieText = config.cookieText
 app.locals.promoMode = promoMode
 app.locals.releaseVersion = 'v' + releaseVersion
 app.locals.serviceName = config.serviceName
 
+// Force HTTPs on production connections
+if (env === 'production' && useHttps === 'true') {
+  app.use(utils.forceHttps)
+}
+
+console.log({'autodata': useAutoStoreData})
+
+if (useAutoStoreData === 'true') {
+  app.use(utils.autoStoreData)
+  app.use(function (req, res, next) {
+    // add nunjucks function to get values, needs to be here as they need access to req.session
+
+    nunjucksAppEnv.addGlobal('checked', function (name, value) {
+      if (req.session.data === undefined) {
+        return ''
+      }
+
+      var storedValue = req.session.data[name]
+
+      if (storedValue === undefined) {
+        return ''
+      }
+
+      if (Array.isArray(storedValue)) {
+        var inArray = storedValue.indexOf(value) !== -1
+        return inArray ? 'checked' : ''
+      } else {
+        return value === storedValue ? 'checked' : ''
+      }
+    })
+
+    next()
+  })
+
+  documentationApp.use(function (req, res, next) {
```

Comment:
The app exists, but routes aren't set up for it.

Later in `server.js` we use `if (useDocumentation) {}` - do you think something similar would be appropriate here?

I don't have a strong preference here - if you think it's not needed that's fine.